solutions/gryphon.md

```diff
@@ -0,0 +1,103 @@
+[![CICD-SEC-3 Dependency Chain Abuse](https://img.shields.io/badge/CICD--SEC--3-Dependency%20Chain%20Abuse-brightgreen)](https://www.cidersecurity.io/top-10-cicd-security-risks/dependency-chain-abuse/?utm_source=github&utm_medium=github_page&utm_campaign=ci%2fcd%20goat_100422)
+
+
+Let's explore what we have when signing in to the compromised GitLab:
+You are the maintainer of *pygryphon* package, there are also public projects *nest-of-gold* and *awesome-app* which we have a read-only access.
+<BR><BR>
+As the instructions of this challenge hint us, let's try to see if we can find any reference to *Flag11*.
+  <BR>
+A simple search tells us it is being referenced by *nest-of-gold* project 
+  <BR><BR><BR>
+
+![grpyhon](../images/gryphon-1.png "grpyhon")
+
+  <BR><BR><BR>
+We can see that *Flag11* is used in a pipeline that ships a container used in production.
+Let's go to this pipeline scheduling, we can see that user named "gryphon" is the owner of this pipeline, and this user is also the owner of the *awesome-app* project's pipeline.
```

Comment:
you can't necessarily determine that the user is an owner you can only know that it has write permissions to trigger the pipeline